.travis.yml

```diff
@@ -3,11 +3,13 @@ language: scala
 before_script:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
-  - sleep 3 # give xvfb some time to start
+  - python -m SimpleHTTPServer 8080 &
+  - sleep 3
```

Comment:
Given that the `scalastyle` took 165.06s in one of the travis test instanced, I would just ignore the `sleep` here.